app/src/main/kotlin/io/homeassistant/companion/android/util/compose/glance/GlanceExt.kt

```diff
@@ -0,0 +1,11 @@
+package io.homeassistant.companion.android.util.compose.glance
```

Comment:
Does this really need a package if it is the only thing? The name of the file is already pretty descriptive.